src/renderers/shaders/ShaderChunk/aomap_fragment.glsl.js

```diff
@@ -6,11 +6,18 @@ export default /* glsl */`
 
 	reflectedLight.indirectDiffuse *= ambientOcclusion;
 
+
 	#if defined( USE_ENVMAP ) && defined( STANDARD )
 
 		float dotNV = saturate( dot( geometryNormal, geometryViewDir ) );
 
-		reflectedLight.indirectSpecular *= computeSpecularOcclusion( dotNV, ambientOcclusion, material.roughness );
+		float specularOcclusion = computeSpecularOcclusion( dotNV, ambientOcclusion, material.roughness );
+
+		reflectedLight.indirectSpecular *= specularOcclusion;
+
+		#if defined( USE_SHEEN ) 
+			sheenSpecularIndirect *= specularOcclusion;
```

Comment:
I had a look at that frostbite paper - I'm not sure I love that AO approximation, but I see what they're trying to do. Still, I think it may be doing the opposite of what we want in the sheen case - I would recommend we just multiply by `ambientOcclusion` directly instead of `specularOcclusion`, at least until we do a serious analysis.

Reply:
I'm inclined to agree.

Also, we should be occluding incoming irradiance -- not reflected light. It may be mathematically equivalent, but it is just odd.

Reply:
Odd yes, but in the spirit of minimally-invasive changes, this seems reasonable.

Reply:
I agree. I'm just making the point, as you think about the problem. :-)